debian/k8s-cni/99_bridge.conf

```diff
@@ -0,0 +1,18 @@
+{
+  "cniVersion": "0.1.0",
+  "name": "kubenet",
```

Comment:
Kubenet doesn't actually provide an overlay. We are working on turning it into a cni plugin. It will be a metaplugin that configures the bridge plugin according to the cidrs passed down by the node controller. This is very usful when running in cloud providers that supported routed networks.